chainer/training/extensions/_snapshot.py

```diff
@@ -4,8 +4,11 @@
 from chainer.utils import argument
 
 
-def snapshot_object(target, filename, savefun=npz.save_npz, **kwargs):
-    """Returns a trainer extension to take snapshots of a given object.
+def snapshot_object(target, filename, **kwargs):
```

Comment:
Are you sure to make a no-compat change by forcing an argument to be a keyword?  The function is publicly available as `chainer.training.extensions.snapshot_object`, while this file `chainer/training/extensions/_snapshot.py` looks private.

Reply:
Thanks for good catch. I knew it was public function and caring default behaviour but less careful about custom savefun case. I added another commit, which might seem strange that the default value of `savefun` being `None`, but it preserves previous behaviour when either `None` or some function is passed.